pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -249,6 +253,37 @@ var (
 	RotationPolicyAlways PrivateKeyRotationPolicy = "Always"
 )
 
+// The type of additional private key format to write
```

Comment:
suggestion: This line doesn't read right when it's serialized into the description for `deploy/crds/crd-certificates.yaml`. 
```suggestion
// The type of additional private key format to write.
```

Reply:
added dot `.`